### PR TITLE
Rewrite and enable service debug test with new service syntax changes

### PR DESCRIPTION
**Pull request by praveennadarajah**
## Purpose
This PR will rewrite and enable service debug test with new service syntax changes.

Fixes: https://github.com/ballerina-platform/ballerina-distribution/issues/1016

### CODE REVIEW

File: devtools-integration-tests/src/test/resources/debug/project-based-tests/service-tests/hello_service.bal [hunk 1]
```diff
@@ -15,17 +15,10 @@
...
-    // Resource functions are invoked with the HTTP caller and the incoming request as arguments.
-    resource function sayHello(http:Caller caller, http:Request req) {
-        // Send a response back to the caller.
-        var result = caller->respond("Greeting, World!");
-        if (result is error) {
-            log:printError("Error sending response", result);
-        }
+service on new http:Listener(9191) {
+    resource function get sayHello(http:Caller caller,
+        http:Request req) returns error? {
```
Comment (NipunaRanasinghe) [inline devtools-integration-tests/src/test/resources/debug/project-based-tests/service-tests/hello_service.bal:21]: Can we merge these two lines (L20 and L21)?

File: devtools-integration-tests/src/test/resources/debug/single-file-tests/hello_service.bal [hunk 2]
```diff
@@ -17,26 +17,21 @@
...
-        if (res is string){
-            payload = res;
-        }
-        test:assertEquals(payload, "Hello Test!", "Service involation test");
+        test:assertEquals(response.getTextPayload(), "Hello, World!", "Service involation test");
     } else {
         test:assertFail(response.toString());
     }
-}
+}
```
Comment (NipunaRanasinghe) [inline devtools-integration-tests/src/test/resources/debug/single-file-tests/hello_service.bal:37]: missing new line

File: devtools-integration-tests/src/test/resources/debug/single-file-tests/hello_service.bal [hunk 3]
```diff
@@ -17,26 +17,21 @@
 import ballerina/http;
 import ballerina/test;
 
-service hello on new http:Listener(9090) {
-    resource function sayHello(http:Caller caller,
+service on new http:Listener(9090) {
+    resource function get sayHello(http:Caller caller,
```
Comment (NipunaRanasinghe) [inline devtools-integration-tests/src/test/resources/debug/single-file-tests/hello_service.bal:21]: Can we merge these two lines (L21 and L22)?

File: devtools-integration-tests/src/test/resources/debug/project-based-tests/service-tests/hello_service.bal (hunk 1 above)
Comment (praveennadarajah) [reply devtools-integration-tests/src/test/resources/debug/project-based-tests/service-tests/hello_service.bal]: Fixed

File: devtools-integration-tests/src/test/resources/debug/single-file-tests/hello_service.bal (hunk 2 above)
Comment (praveennadarajah) [reply devtools-integration-tests/src/test/resources/debug/single-file-tests/hello_service.bal]: Fixed

File: devtools-integration-tests/src/test/resources/debug/single-file-tests/hello_service.bal (hunk 3 above)
Comment (praveennadarajah) [reply devtools-integration-tests/src/test/resources/debug/single-file-tests/hello_service.bal]: Fixed

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**